"""
Exception raised when the app fails to process the request and due to an error it couldn't handle correctly
"""


class InternalException(BaseException):
    """
    InternalException
    """
    pass
